pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <artifactId>mod-circulation-storage</artifactId>
   <groupId>org.folio</groupId>
-  <version>16.1.0-SNAPSHOT</version>
+  <version>16.1.1-SNAPSHOT</version>
```

Comment:
No need to change module version here, but interface version needs to be changed

Reply:
Updated request-storage to 5.1